docs/sdks/flutter/GETTING_STARTED.md

```diff
@@ -0,0 +1,115 @@
+
+### Android
+For **Android** add your app <u>name</u> and <u>package name</u>, Your package name is generally the applicationId in your app-level build.gradle file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.
+
+#### iOS
+
+The Appwrite SDK uses ASWebAuthenticationSession on iOS 12+ and SFAuthenticationSession on iOS 11 to allow OAuth authentication. You have to change your iOS Deployment Target in Xcode to be iOS >= 11 to be able to build your app on an emulator or a real device.
+
+1. In Xcode, open Runner.xcworkspace in your app's ios folder.
+2. To view your app's settings, select the Runner project in the Xcode project navigator. Then, in the main view sidebar, select the Runner target.
+3. Select the General tab.
+4. In Deployment Info, 'Target' select iOS 11.0
+
+### Android
+In order to capture the Appwrite OAuth callback url, the following activity needs to be added to your [AndroidManifest.xml](https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml). Be sure to relpace the **[PROJECT_ID]** string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.
+
+```xml
+<manifest>
+    <application>
+        <activity android:name="com.linusu.flutter_web_auth.CallbackActivity" >
+            <intent-filter android:label="flutter_web_auth">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="appwrite-callback-[PROJECT_ID]" />
+            </intent-filter>
+        </activity>
+    </application>
+</manifest>
+```
+
+#### Web
+Appwrite 0.7, and the Appwrite Flutter SDK 0.3.0 have added support for Flutter Web. To build web apps that integrate with Appwrite successfully, all you have to do is add a web platform on your Appwrite project's dashboard and list the domain your website will use to allow communication to the Appwrite API.
+
+### Flutter Web Cross-Domain Communication & Cookies
+While running Flutter Web, make sure your Appwrite server and your Flutter client are using the same top-level domain and the same protocol (HTTP or HTTPS) to communicate. When trying to communicate between different domains or protocols, you may receive HTTP status error 401 because some modern browsers block cross-site or insecure cookies for enhanced privacy. In production, Appwrite allows you set multiple [custom-domains](https://appwrite.io/docs/custom-domains) for each project.
+
+### Init your SDK
+
+<p>Initialize your SDK code with your project ID, which can be found in your project settings page.
+
+```dart
+import 'package:appwrite/appwrite.dart';
+Client client = Client();
+
+
+client
+.setEndpoint('https://localhost/v1') // Your Appwrite Endpoint
```

Comment:
Should this code be formatted with indentation? Not sure if that happens in the markdown file or somewhere else.